<?xml version="1.0" encoding="utf-8"?>

<launch>
  <!-- Parameters -->
  <arg name="power_gain" default="0.1"/>
  <arg name="speed_gain" default="0.1"/>
  <arg name="omega_gain" default="0.1"/>

  <!-- Start Motors -->
  <node pkg="grmi_suitcase" type="motors_control.py" name="motors_control" output="screen">
    <param name="power_gain"           value="$(arg power_gain)" />
    <param name="speed_gain"           value="$(arg speed_gain)" />
    <param name="omega_gain"           value="$(arg omega_gain)" />
  </node>

</launch>
